README.md

```diff
@@ -1,32 +1,144 @@
-
-### Terminology
-Described DataRobot functionality is known as `Custom Model`, so words `custom model` and `user model` may be interchangeably used.
-Also `custom model directory` and `code directory` mean the same entity.
-
 
 ## Content  
-- [model templates](model_templates) - contains templates for building and deploying custom models in DataRobot.
-- [environment templates](public_dropin_environments) - contains templates of base environments used in DataRobot. Dependency requirements can be applied to the base to create runtime environment for custom models.
-- [user models runner](custom_model_runner) - **drum** - a tool that helps to assemble, test, and run custom model. For more information about how to run drum, check out the [pypi docs](https://pypi.org/project/datarobot-drum/)
-
-## Custom Model Templates
-This repository contains templates for building and deploying custom models in DataRobot.
-
-Custom Inference Models are models that are trained outside of DataRobot. Once they're uploaded to DR, they are deployed straight to a DR Deployment, and tracked with Model Monitoring and Management.
-
-Custom Training Models are in active development. They include a `fit()` function, and can be trained on the Leaderboard, benchmarked against DR AutoML models, and get access to our full set of automated insights. Check out [The quickrun readme here](QUICKSTART-FOR-TRAINING.md)
+1. [What is this repository?](#what_is_it)
+2. [Quickstart and examples](#quickstart)
+3. [Assembling inference model code folder](#inference_model_folder)
+4. [Custom Model Templates](#custom_model_templates)
+5. [Custom Environment Templates](#custom_environment_templates)
+6. [Custom Model Runner (drum)](#custom_model_runner)
+4. [Contribution & development](#contribution_development)
+
+## What is this repository? <a name="what_is_it"></a>
+The **DataRobot User Models** repository contains information and tools for assembling,
+debugging, testing, and running your training and inference models with DataRobot.
+
+#### Terminology
+This repository address the DataRobot functionality known as `custom models`. The terms `custom model` and `user model` can be used interchangeably, as can `custom model directory` and `code directory`.
+
+## Quickstart <a name="quickstart"></a>
+The following example shows how to use the [**drum**](https://github.com/datarobot/datarobot-user-models/tree/master/custom_model_runner) tool to make predictions on an [sklearn model](model_templates/inference/python3_sklearn)
+1. Clone the repository
+2. Create a virtual environment: `python3 -m virtualenv <dirname for virtual environment>`
+3. Activate the virtual environment: `source <dirname for virtual environment>/bin/activate`
+4. cd into the repo: `cd datarobot-user-models`
+5. Install the required dependencies: `pip install -r public_dropin_environments/python3_sklearn/requirements.txt`
+6. Install datarobot-drum: `pip install datarobot-drum`
```

Comment:
all of this could be removed with my plan!

Reply:
What is your plan?

Reply:
to build a docker image locally and run drum in it